.drone.yml

```diff
@@ -17,7 +17,7 @@ clone:
 
 steps:
   - name: compile
-    image: hub.scireum.com/scireum/sirius-build-jdk16:latest
+    image: hub.scireum.com/scireum/sirius-build-jdk17:latest
```

Comment:
In de Konv steht nix mehr von hub.scireum.com?

Reply:
Ja, ist gefixed